src/smjsindustry/finance/nlp_score_type.py

```diff
@@ -27,7 +27,7 @@ class NLPScoreType:
     to perform NLP scoring using SageMaker JumpStart Industry's internal word lists.
 
     A single ``NLPScoreType`` or a list of ``NLPScoreTypes`` is required
-    when initializing an :class:`~smjsindustry.NLPScorerConfig`. Passing the data required by
+    when initializing a :class:`~smjsindustry.NLPScorerConfig`. Passing the data required by
```

Comment:
The doc renders like the following image:
<img width="712" alt="Screen Shot 2021-09-29 at 3 03 30 PM" src="https://user-images.githubusercontent.com/21208346/135354891-16f77a95-cd6c-4e4a-9aed-abd5d11eebd1.png">

I think it should still be "an"?

Reply:
Oh yes, sorry! Didn't know that was how it rendered.